mifospay/src/main/java/org/mifos/mobilewallet/mifospay/editprofile/ui/EditProfileActivity.java

```diff
@@ -433,8 +434,8 @@ public void onPause() {
 
         cancelProgressDialog();
 
-        if (dialogBox != null) {
-            dialogBox.dismiss();
+        if (dialogBox.alertDialog != null) {
```

Comment:
instead of doing it like this, add the alertDialog check inside the dialogbox itself.
In DialogBox, you can do-
```
public void dismiss() {
      if (alertDialog != null) {
          alertDialog.dismiss();
      }
    }
```

Reply:
@jatinsachdevame not resolved yet

Reply:
Also, I have just merged this PR https://github.com/openMF/mobile-wallet/pull/644 so this can be closed now